CHANGELOG.md

```diff
@@ -40,7 +40,7 @@ and includes an additional section for migration notes.
   - database_app_user_pw-> db_user_password (*new*)
 - These are the new variables added:
   - db_admin_username (defaults to "postgres")
-  - db_host_endpoint
+  - db_host_endpoint (Set in variables.tf to your RDS Cluster's endpoint, similar to "PREFIX-cumulus-db.cluster-000000000000.us-west-2.rds.amazonaws.com")
```

Comment:
Maybe add that it is a required variable that needs to be set?

Reply:
Good point. How's this?